notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<p>* What is a "positive" (user, movie) pair? Not sure I understand the meaning of "positive". Do we have "negative" pairs?</p><p>* Explain what "FactorizedTopK" does (what is the input, what's is computed, etc.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Did you explain the above? Not seeing it. </p>